packages/medusa/src/api/routes/admin/regions/update-region.ts

```diff
@@ -44,6 +46,9 @@ import { EntityManager } from "typeorm"
  *           tax_rate:
  *             description: "The tax rate to use on Orders in the Region."
  *             type: number
+ *          includes_tax:
+ *             description: "[EXPERIMENTAL] Is the region tax inclusive or not"
```

Comment:
suggestion: Rephrase to improve semantics
```suggestion
 *             description: "[EXPERIMENTAL] Tax included in prices in region"
```